EventKit/eventkit-cloud
# Updating dependency versions

# Description of Improvement

Updates dependencies highlighted by dependabot to handle any security vulnerabilities found in versions.

## How to test

This will require a full rebuild of conda dependencies and images.
Verify tests pass
verify export works with multiple provider types

## Quality Assurance

The following items have been updated:


## Screenshots

:smile:
